packages/xo-web/src/common/xo/index.js

```diff
@@ -1245,7 +1248,32 @@ export const migrateVm = (vm, host) =>
         _('migrateVmNoTargetHostMessage')
       )
     }
-    return _call('vm.migrate', { vm: vm.id, ...params })
+
+    return _call('vm.migrate', { vm: vm.id, ...params }).catch(err =>
+      confirm({
```

Comment:
IMO, we should only ask for a retry if the error is the CPU difference (which is the [original reason why we force migrations](https://github.com/vatesfr/xen-orchestra/issues/2136#issue-226402924)).